container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -545,6 +574,8 @@ impl pallet_evm::Config for Runtime {
     type OnChargeTransaction = ();
     type OnCreate = ();
     type FindAuthor = FindAuthorTruncated<Aura>;
+    type Timestamp = Timestamp;
+    type WeightInfo = ();
```

Comment:
fyi WeightInfo for pallet-evm has been recently added to frontier, so you don't forget to update this once you pin a more recent version.